v1.1/orchestrate-cockroachdb-with-docker-swarm-insecure.md

```diff
@@ -197,57 +201,31 @@ $ sudo docker network create --driver overlay cockroachdb
 
     {{site.data.alerts.callout_success}}The service definitions tell the CockroachDB nodes to log to <code>stderr</code>, so if you ever need access to a node's logs for troubleshooting, use <a href="https://docs.docker.com/engine/reference/commandline/logs/"><code>sudo docker logs &lt;container id&gt;</code></a> from the instance on which the container is running.{{site.data.alerts.end}}
 
-4. Remove the first service and recreate it again with the `--join` flag to ensure that, if the first node restarts, it will rejoin the original cluster via the the second service, `cockroachdb-2`, instead of initiating a new cluster:
+3. Now all the CockroachDB nodes are running, but we still have to explicitly tell them to initialize a new cluster together. To do so, use the `sudo docker run` command to run the `cockroach init` command against one of the nodes. The `cockroach init` command will initialize the cluster, bringing it into a usable state.
 
     {% include copy-clipboard.html %}
     ~~~ shell
-    $ sudo docker service rm cockroachdb-1
+    $ sudo docker run -it --rm --network=cockroachdb cockroachdb/cockroach:{{page.release_info.version}} ./cockroach init --host=cockroachdb-1 --insecure
     ~~~
 
-    {% include copy-clipboard.html %}
-    ~~~ shell
-    $ sudo docker service create \
-    --replicas 1 \
-    --name cockroachdb-1 \
-    --hostname cockroachdb-1 \
-    --network cockroachdb \
-    --mount type=volume,source=cockroachdb-1,target=/cockroach/cockroach-data,volume-driver=local \
-    --stop-grace-period 60s \
-    --publish 8080:8080 \
-    cockroachdb/cockroach:{{page.release_info.version}} start \
-    --join=cockroachdb-2:26257 \
-    --logtostderr \
-    --insecure
-    ~~~
 
 ## Step 6. Use the built-in SQL client
 
-1. On any instance, use the `sudo docker ps` command to get the ID of the container running the CockroachDB node:
-
-    {% include copy-clipboard.html %}
-    ~~~ shell
-    $ sudo docker ps | grep cockroachdb
-    ~~~
-
-    ~~~
-    9539871cc769        cockroachdb/cockroach:{{page.release_info.version}}   "/cockroach/cockroach"   2 minutes ago        Up 2 minutes         8080/tcp, 26257/tcp   cockroachdb-1.1.0wigdh8lx0ylhuzm4on9bbldq
-    ~~~
-
-2. Use the `sudo docker exec` command to open the built-in SQL shell in interactive mode inside the container:
+1. Use the `sudo docker run` command to run a new container attached to the CockroachDB network and connect to the cluster:
```

Comment:
I think it's helpful to call out that we're launching the built-in SQL shell, but I see that you removed that language. Do you feel that "connect to the cluster" conveys the same thing?

Reply:
You're right, being more specific about how we're connecting is better. I've changed these back.